Signal/src/ViewControllers/Utils/NeverClearView.swift

```diff
@@ -0,0 +1,12 @@
+// Created by Fredrik Lillejordet on 04.03.2018.
+// Copyright © 2018 Open Whisper Systems. All rights reserved.
```

Comment:
Please run the Scripts/hooks/post-commit to get the right headers (or copy from another file)

Reply:
✅ Done!